CHANGELOG.md

```diff
@@ -1,33 +1,33 @@
 # Vx.x.x (Pre-release)
 
-## Release Hightlights
+## Release Highlights
 
 ## Important Notes
 
 - [#453](https://github.com/oauth2-proxy/oauth2-proxy/pull/453) Responses to endpoints with a proxy prefix will now return headers for preventing browser caching.
 
 ## Breaking Changes
 
-- Migration from Pusher to independent org may have introduced breaking changes for your environment.
+- [#464](https://github.com/oauth2-proxy/oauth2-proxy/pull/464) Migration from Pusher to independent org may have introduced breaking changes for your environment.
   - See the changes listed below for PR [#464](https://github.com/oauth2-proxy/oauth2-proxy/pull/464) for full details
   - Binaries renamed from `oauth2_proxy` to `oauth2-proxy`
-- [#440](https://github.com/oauth2-proxy/oauth2-proxy/pull/440) Switch Azure AD Graph API to Microsoft Graph API (@johejo)
+- [#440](https://github.com/oauth2-proxy/oauth2-proxy/pull/440) Switch Azure AD Graph API to Microsoft Graph API
```

Comment:
How come we removed the username form this one?

Reply:
We've never put the names on breaking changes previously, the names are normally tagged in the changes since section.

I think this is consistent with the rest of the changelog